server/src/pouch-db-app.ts

```diff
@@ -54,10 +56,46 @@ class PouchDbApp extends AppBase {
    * https://github.com/pouchdb/pouchdb-server
    */
   private addPouchRoutes(): void {
+    // If VM lives at non-root prefix, works but fauxton not fully working yet
+    const urlPrefix = process.env[VM_URL_PREFIX] || '/';
+    const config = {
+      mode: 'fullCouchDB', inMemoryConfig: true,
+      "httpd": {
+        "enable_cors": true
+      },
+      "cors": {
+        "origins": "https://skarabrae.org",
```

Comment:
General comment, the server code has a mix of config files and config in TS now (here's one example but there's also lots of GCP specific bits in TS constants). Likely worth moving everything to config YAML or otherwise at some point, I saw you had similar comment about doing things in JSON instead elsewhere. Perhaps worth a follow up issue filing.

Reply:
Created issue